generators/server/templates/src/main/resources/config/_application-dev.yml

```diff
@@ -256,6 +263,23 @@ jhipster:
             time-to-live-seconds: 3600 # By default objects stay 1 hour in the cache
             max-entries: 100 # Number of objects in each cache entry
         <%_ } _%>
+        <%_ if (hibernateCache == 'infinispan') { _%>
+        infinispan:
+            config-file: default-configs/default-jgroups-udp.xml
```

Comment:
Yes (comes from jar) - default implementation. Users can customise and include local file as well by specifying the path here.